### PR TITLE
bug fix: upgrade arangosync version during DC2DC deployment upgrade

### PR DESCRIPTION
**Pull request by naushniki**

### CODE REVIEW

**Comment by naushniki** [comment]
https://jenkins.arangodb.biz/view/RTA/job/Upgrade-3.11-to-devel.Linux/543/

File: release_tester/arangodb/sync.py [hunk 1]
```diff
@@ -48,6 +48,7 @@ def run_syncer(self):
     def replace_binary_for_upgrade(self, new_install_cfg):
         """set the new config properties"""
```
Comment (dothebart) [inline release_tester/arangodb/sync.py:49]: imho
```
    def set_from(self, other_cfg):
        """copy constructor"""
```
should be used here?

**Review by dothebart — APPROVED** [review]
please use ` set_from()`  - else LGTM.

**Comment by naushniki** [comment]
https://jenkins.arangodb.biz/view/RTA/job/Upgrade-3.9-to-3.10.Linux/1105/

**Pull request merged**